app/scenes/Login/index.js

```diff
@@ -164,7 +193,12 @@ function Login({ location }: Props) {
 
         {isCreate && (
           <Note>
-            Already have an account? Go to <Link to="/">login</Link>.
+            <Trans
+              defaults="Already have an account? Go to <link>."
+              components={{
+                link: <Link to="/">login</Link>,
+              }}
+            />
```

Comment:
```suggestion
            <Trans>
              Already have an account? Go to <Link to="/">login</Link>.
            </Trans>
```

This syntax doesn't work I'm afraid

![image](https://user-images.githubusercontent.com/380914/126192892-99d77e21-3874-4267-87aa-685e502c494e.png)